packages/mgt/src/components/mgt-people-picker/mgt-people-picker.ts

```diff
@@ -676,12 +676,16 @@ export class MgtPeoplePicker extends MgtTemplatedComponent {
         people = [];
         if (this.type === PersonType.person || this.type === PersonType.any) {
           try {
-            people = (await findPeople(graph, input, this.showMax)) || [];
+            if (this.groupId) {
```

Comment:
If `this.type === "group"` or `this.type === "any"`, the following if statement (line 705) will do a group search outside of the group-id. Could `findPeopleFromGroup` be updated to filter groups? If so, I recommend doing a similar logic there as well

Reply:
Right now, the type is a `PersonType` but I'm wondering if we should open it up to become a `DynamicPersonType`? This would allow to use this enum un both `graph.users` and in `graph.people`. Thoughts?

Reply:
Did not need it. Kept it that way finally!